hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -153,9 +153,16 @@
 
     public GrailResource() {}
 
+    /**
+     * If the PRIVATE_OVERPASS_URL variable is set to a value then return true
+      */
+    private boolean isPrivateOverpassActive() {
```

Comment:
would be nice to use this across in the future so this is how we decide if using private api or not

Reply:
Sure, could be a public static method